code/game/machinery/autolathe_datums.dm

```diff
@@ -330,35 +330,246 @@
 	path = /obj/item/weapon/syringe_cartridge
 	category = "Arms and Ammunition"
 
+////////////////
+/*Ammo casings*/
+////////////////
+
 /datum/autolathe/recipe/shotgun_blanks
-	name = "ammunition (shotgun, blank)"
+	name = "ammunition (12g, blank)"
 	path = /obj/item/ammo_casing/shotgun/blank
 	category = "Arms and Ammunition"
 
 /datum/autolathe/recipe/shotgun_beanbag
-	name = "ammunition (shotgun, beanbag)"
+	name = "ammunition (12g, beanbag)"
 	path = /obj/item/ammo_casing/shotgun/beanbag
 	category = "Arms and Ammunition"
 
 /datum/autolathe/recipe/shotgun_flash
-	name = "ammunition (shotgun, flash)"
+	name = "ammunition (12g, flash)"
 	path = /obj/item/ammo_casing/shotgun/flash
 	category = "Arms and Ammunition"
 
-/datum/autolathe/recipe/magazine_rubber
-	name = "ammunition (.45, rubber)"
-	path = /obj/item/ammo_magazine/c45m/rubber
+/datum/autolathe/recipe/shotgun
+	name = "ammunition (12g, slug)"
+	path = /obj/item/ammo_casing/shotgun
+	hidden = 1
+	category = "Arms and Ammunition"
+
+/datum/autolathe/recipe/shotgun_pellet
+	name = "ammunition (12g, pellet)"
+	path = /obj/item/ammo_casing/shotgun/pellet
+	hidden = 1
+	category = "Arms and Ammunition"
+
+/datum/autolathe/recipe/stunshell
+	name = "ammunition (stun cartridge, shotgun)"
+	path = /obj/item/ammo_casing/shotgun/stunshell
+	hidden = 1
+	category = "Arms and Ammunition"
+
+//////////////////
+/*Ammo magazines*/
+//////////////////
+
+// These should always be /empty! The idea is to fill them up manually with ammo clips.
+
+/datum/autolathe/recipe/pistol_5mm
+	name = "pistol magazine (5mm)"
+	path = /obj/item/ammo_magazine/c5mm
+	category = "Arms and Ammunition"
+	hidden = 1
+
+/datum/autolathe/recipe/smg_5mm
+	name = "top-mounted magazine (5mm)"
+	path = /obj/item/ammo_magazine/c5mmt
+	category = "Arms and Ammunition"
+	hidden = 1
+
+/datum/autolathe/recipe/pistol_45
+	name = "pistol magazine (.45)"
+	path = /obj/item/ammo_magazine/c45m/empty
+	category = "Arms and Ammunition"
+
+/datum/autolathe/recipe/pistol_45uzi
+	name = "uzi magazine (.45)"
+	path = /obj/item/ammo_magazine/c45uzi/empty
+	category = "Arms and Ammunition"
+	hidden = 1
+
+/datum/autolathe/recipe/tommymag
+	name = "Tommygun magazine (.45)"
+	path = /obj/item/ammo_magazine/tommymag/empty
+	category = "Arms and Ammunition"
+	hidden = 1
+
+/datum/autolathe/recipe/tommydrum
+	name = "Tommygun drum magazine (.45)"
+	path = /obj/item/ammo_magazine/tommydrum/empty
+	category = "Arms and Ammunition"
+	hidden = 1
+
+/datum/autolathe/recipe/pistol_9mm
+	name = "pistol magazine (9mm)"
+	path = /obj/item/ammo_magazine/mc9mm/empty
+	category = "Arms and Ammunition"
+
+/datum/autolathe/recipe/smg_9mm
+	name = "top-mounted magazine (9mm)"
+	path = /obj/item/ammo_magazine/mc9mmt/empty
+	category = "Arms and Ammunition"
+
+/datum/autolathe/recipe/smg_10mm
+	name = "SMG magazine (9mm)"
```

Comment:
No, it's a 10mm magazine. The 20rnd syndie SMG mag is here, but it's named wrongly.